src/executor.cpp

```diff
@@ -1064,18 +1068,19 @@ types::ReturnStatus
             case 70:
                 sendFuncNumToPhyp(funcNum);
                 break;
-
-            default:
-                std::cerr << "Given function number "
-                          << static_cast<int>(funcNum) << " is not supported."
-                          << std::endl;
-                executionStatus = std::make_tuple(false, "", "");
         }
     }
+    catch (const sdbusplus::exception_t& e)
+    {
+        isExternallyTriggered = false;
+        std::cerr << e.what() << std::endl;
```

Comment:
Along with e.what(), add the details that you had in the cerr previously. Will be helpful in debugging.